pkg/modulebuilder/buildtype/buildtype.go

```diff
@@ -0,0 +1,22 @@
+package buildtype
+
+import "github.com/asaskevich/govalidator"
```

Comment:
When you told me I thought you were talking about this https://github.com/go-playground/validator

Reply:
Yeah, used the wrong one! Feel free to change it in a new PR